### PR TITLE
add keyboard interrupt to create_napp

### PR DESCRIPTION
**Pull request by slinksoft**
The Capstone 1 team at FIU attempted to solve issue #212  , in which we simply solved by catching a KeyboardInterrupt exception on create_napp()

This is a redo of the 2nd part of pull request #284 . We solved this issue during the beginning of the semester. At the time, our team didn't know how to use Github properly and how to go about creating pull requests. This time, we created a separate branch that changes to fix only issue #212 .

### CODE REVIEW

**Comment by slinksoft** [comment]
@hdiogenes , we are getting the same error as last time for text_napps, but we do not understand how adding a simple try-except to handle a KeyboardInterrupt is causing the test to fail. Any advice on this?

**Comment by slinksoft** [comment]
We were able to fix the errors causing test_napps to fail and tested our implementation. It works! Now, we are going to resolve the pylint errors and turn this draft PR into a PR.

File: kytos/utils/napps.py [hunk 1]
```diff
@@ -335,13 +336,25 @@ def create_napp(cls, meta_package=False):
         print('--------------------------------------------------------------')
         print('')
         while not cls.valid_name(username):
-            username = input('Please, insert your NApps Server username: ')
+            try:
+                username = input('Please, insert your NApps Server username: ')
+            except KeyboardInterrupt:
+                print("\nUser cancelled napps creation.")
```
Comment (cmagnobarbosa) [inline kytos/utils/napps.py:342]: Hello @slinksoft, thanks for your contribution. I have a small suggestion, maybe you can put this message in a variable, avoiding duplicating this message in `try except`.

File: kytos/utils/napps.py [hunk 2]
```diff
@@ -335,13 +336,25 @@ def create_napp(cls, meta_package=False):
...
+            try:
+                napp_name = input('Please, insert your NApp name: ')
+            except KeyboardInterrupt:
+                print("\nUser cancelled napps creation.")
+                sys.exit(0)
+        try:
+            description = input('Please, insert a brief description for your'
+                                'NApp [optional]: ')
+        except KeyboardInterrupt:
+            print("\nUser cancelled napps creation.")
```
Comment (cmagnobarbosa) [inline kytos/utils/napps.py:355]: same here

File: kytos/utils/napps.py (hunk 1 above)
Comment (cmagnobarbosa) [inline kytos/utils/napps.py:342]: Hi @slinksoft, thanks for your contribution. I have a small suggestion, maybe you can put this message, line 342, in a variable, avoiding duplicating this message in `try except`.``

File: kytos/utils/napps.py [hunk 3]
```diff
@@ -335,13 +336,25 @@ def create_napp(cls, meta_package=False):
...
+            except KeyboardInterrupt:
+                print("\nUser cancelled napps creation.")
+                sys.exit(0)
 
         while not cls.valid_name(napp_name):
-            napp_name = input('Please, insert your NApp name: ')
+            try:
+                napp_name = input('Please, insert your NApp name: ')
+            except KeyboardInterrupt:
+                print("\nUser cancelled napps creation.")
```
Comment (cmagnobarbosa) [inline kytos/utils/napps.py:349]: If you accept the suggestion, use the variable here.

Comment (slinksoft) [reply kytos/utils/napps.py:349]: Ah yes. That shouldn't be a problem. Will refractor it as soon as I can!

**Review by cmagnobarbosa — APPROVED** [review]

**Review by josemauro — APPROVED** [review]

**Comment by josemauro** [comment]
This PR is being closed and the modifications are being adjusted in https://github.com/kytos/kytos-utils/pull/318

**Pull request closed without merging**